idd/Energy+.idd.in

```diff
@@ -54105,6 +54105,26 @@ AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl,
        \type object-list
        \object-list QuadraticCurves
        \object-list UniVariateTables
+  A8 , \field Master Thermostat Priority Control Type
+       \type choice
+       \key LoadPriority
+       \key ZonePriority
+       \key ThermostatOffsetPriority
+       \key MasterThermostatPriority
+       \key Scheduled
+       \default MasterThermostatPriority
+       \note Choose a thermostat control logic scheme. If these control types fail to control zone
+       \note temperature within a reasonable limit, consider using multiple VRF systems
+  A9 , \field Zone Name for Master Thermostat Location
+       \type object-list
+       \object-list ZoneNames
+       \note This field is required if Master Thermostat Priority Control Type is MasterThermostatPriority
+       \note Enter the name of the zone where the master thermostat is located
+  A10, \field Thermostat Priority Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note this field is required if Master Thermostat Priority Control Type is Scheduled
+       \note Schedule values of 0 denote cooling, 1 for heating, and all other values disable the system
```

Comment:
Ummmmmm, OK.  So this can't go in the bug fix release then.  

@rongpengzhang Is there a way to extract out just the bug fix portion without these idd changes?  Or are they coupled?  

FYI @mjwitte